config/onprem-iso-fcc.yaml

```diff
@@ -80,15 +101,18 @@ storage:
                 DB_USER=admin
                 DB_PASS=admin
                 DB_NAME=installer
-                SERVICE_BASE_URL=http://assisted-api.local.openshift.io:8080
+                SERVICE_BASE_URL=https://192.168.122.5:8443
```

Comment:
IP address shouldn't be hard coded.

Reply:
This is more of a placeholder right now to indicate the dns url doesn't work until /etc/host changes.